Detections/QualysVM/HighNumberofVulnDetected.yaml

```diff
@@ -0,0 +1,27 @@
+id: be52662c-3b23-435a-a6fa-f39bdfc849e6
+name: High Number of Urgent Vulnerabilities Detected
+description: |
+  'This Creates an incident when a host has a high number of Urgent, severity 5, vulnerabilities detected.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CustomConnector
+    dataTypes: 
+      - QualysHostDetection_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
```

Comment:
Tactics and Techniques are required - https://attack.mitre.org/techniques/enterprise/

Reply:
corrections have been made and pushed.